src/en.json

```diff
@@ -37,20 +37,7 @@
             "collectibles": "Collectibles",
             "created": "Created",
             "details": "Details",
-            "marketplaces": {
-                "chessNft": {
-                    "description": "Unique chess collections",
-                    "title": "Chess NFT"
-                },
-                "grandbazar": {
-                    "description": "Rare digital items",
-                    "title": "Grandbazar"
-                },
-                "thirdPlace": {
-                    "description": "Real-world art objects",
-                    "title": "Third Place"
-                }
-            },
+            "empty": "No collections yet",
```

Comment:
@AnnStepanova could you please provide more appropriate text for the empty Collectibles screen?